movie_api/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for movie_api project.
+
+Generated by 'django-admin startproject' using Django 4.2.9.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-*b)6#%6c-15wcd9__(h8k)%&jboo2#s$0w2%on0a&jj4xd%#ir"
+)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+    "cinema",
+    "rest_framework",

```

Comment:
Your custom apps should be last in the list

Reply:
got it, I thought about it also, but I was checking the django rest framework documentation and it made me think it should be last because of how it's shown here without ```...``` after "rest_framework":
![chrome_SJIZ5vq3mL](https://github.com/mate-academy/py-movie-api/assets/13889031/8c32e026-4dcb-4b2f-bfcf-01d6b61da359)

Reply:
I got you 🙌